src/Microsoft.TestPlatform.Extensions.HtmlLogger/HtmlTransformer.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
```

Comment:
Follow the pattern as in the other files, using statements should be inside the namespace.